### ISSUE TITLE
Update 5 to 6 Migration Instructions

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#1671]
### Description
Once the migration task is finished, documentation needs to be updated to be comprehensive.

### Done Looks Like
Current documentation for running the Avalon 5->Avalon 6 migration task is evaluated and updated as needed.

### Note from Leah on previous issue
Still some problems with rolemaps, maybe something with the rolemap cache needing to be cleared. Once that bug is figured out, we may need to update documentation.

### Previous Issue
#1416

**Comment by mcwhitaker** [comment]
@joncameron we have concluded that there were no problems with the rolemaps and this enabled us to check the last item on #1416 . 

I am not sure what this new ticket is for.
